var/da/da_obs/da_fill_obs_structures_radar.inc

```diff
@@ -40,8 +40,14 @@ subroutine da_fill_obs_structures_radar(iv, ob)
       do n = 1, iv%info(radar)%nlocal
          do k = 1, iv%info(radar)%levels(n)
             ! Copy observation variables:
-            ob % radar(n) % rv(k) = iv % radar(n) % rv(k) % inv
-            ob % radar(n) % rf(k) = iv % radar(n) % rf(k) % inv
+            !ob % radar(n) % rv(k) = iv % radar(n) % rv(k) % inv
```

Comment:
There is no else branch to support the existing behavior. Has this been tested on other radar schemes to indicate bit-for-bit results?

Reply:
This just adds some QC check, nothing to do with other schemes.